frontend/src/pages/BYONImages/UpdateImageModal.tsx

```diff
@@ -39,13 +39,13 @@ export const UpdateImageModal: React.FC<UpdateImageModalProps> = ({
 }) => {
   const [name, setName] = React.useState<string>(image.name);
   const [description, setDescription] = React.useState<string>(
-    image.description != undefined ? image.description : '',
+    image.description !== undefined ? image.description : '',
```

Comment:
Note sure if `image.description` can ever equal `null`, but this change has potentially different outcomes. I'd simplify this to the following (applies to other changes as well):
```suggestion
    image.description ?? '',
```